ui/src/app/sensors/sensor-details.tsx

```diff
@@ -59,12 +60,16 @@ export function SensorDetails({match, location, history}: RouteComponentProps<an
         services.sensor
             .get(name, namespace)
             .then(setSensor)
```

Comment:
```suggestion
            .then(resetSensor)
```

shouldn't this also use `resetSensor`? this is the initial retrieval, so it would make sense to set the initial one here.
also I think this is the odd one out of all the other changed components

Reply:
You're right. Thanks for the catch! Fixed in 1ef4958e4469e8ebd41719a81a45201385e7e476